service-deployment/bootstrap/prometheus/node-exporter-ds.yaml.template

```diff
@@ -52,26 +53,44 @@ spec:
           readOnly: true
         name: node-exporter
         args:
-          - '--collector.textfile.directory={{ clusterinfo[ 'dataPath' ] }}/prometheus'
+          - '--collector.textfile.directory=/datastorage/prometheus'
         ports:
         - containerPort: {{ clusterinfo['prometheusinfo']['node_exporter_port'] }}
           hostPort: {{ clusterinfo['prometheusinfo']['node_exporter_port'] }}
           name: scrape
       - image: {{ clusterinfo['dockerregistryinfo']['prefix'] }}gpu-exporter:{{ clusterinfo['dockerregistryinfo']['docker_tag'] }}
+        imagePullPolicy: Always
```

Comment:
>Always [](start = 25, length = 6)

why do we need this?

Reply:
yundong's issue https://github.com/Microsoft/pai/issues/460

Reply:
refresh local image